chaos-crds/values.yaml

```diff
@@ -0,0 +1,10 @@
+global:
+  clusterScope:
+    enabled: false
+  autoUpgrade:
+    enabled: true
+  infraId: "1234"
```

Comment:
Sample value/place holder can be added as comment don't add it in values if it is not a real value